app/views/users/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>List of Users</h1>
+<div class="container">
+  <h2>Users</h2>
+  <% @users.each do |user| %>
+    <%= link_to user_path(user) do %>
+      <div class="user-list">
```

Comment:
- [ ] Your app appears to be broken upon trying to implement the previously requested changes. Users details as well as other pages are showing blank, even though sample data are correctly entered in the database. Kindly ensure to improve upon this aspect:

![image](https://github.com/saluumaa/blog-app/assets/12745474/ce026067-fdc6-4fd0-b754-3ce9c7051a00)

![image](https://github.com/saluumaa/blog-app/assets/12745474/367e9296-1819-4a34-805e-22907fad8f36)

![image](https://github.com/saluumaa/blog-app/assets/12745474/b92a1bfc-dc1c-46e7-a548-ad38454dbaea)

![image](https://github.com/saluumaa/blog-app/assets/12745474/982ac7fa-e4f8-4041-b0b6-2918a2650318)

![image](https://github.com/saluumaa/blog-app/assets/12745474/b511a91b-4493-4adb-bfd5-d28e2d0033fd)